app/assets/stylesheets/layouts/_main-content.scss

```diff
@@ -6,3 +6,10 @@
 .action-links {
   margin-top: 1em;
 }
+
+.voted_img {
```

Comment:
We like to use hyphens for class names. The key here is that we use hyphens in this project and consistency is great.